programs/degen-pools/src/pools.rs

```diff
@@ -87,6 +89,45 @@ pub fn set_winning_option(ctx: Context<UpdatePool>, winning_option: Pubkey) -> R
     Ok(())
 }
 
+pub fn fund_pool<'c: 'info, 'info>(
+    ctx: Context<'_, '_, 'c, 'info, FundPool<'info>>,
```

Comment:
What is the need for the initial arguments? Can't we follow the same pattern as https://github.com/Degen-Markets/degen-markets-solana/pull/12/files#diff-e4fe02d9ea4178a1c23a95acc10402633301397989504c69116d932a115bfefeR79 ?

Reply:
Yes initially it was written like that, but there was a lifetime issue. I'm following [this stack overflow answer](https://solana.stackexchange.com/a/8450/24881), and annotating that the `remaining_accounts` has at least the same lifetime as our `FundPool` Instruction Struct accounts